src/utils/jump_utils.jl

```diff
@@ -44,57 +45,162 @@ function fix_parameter_value(input::JuMP.VariableRef, value::Float64)
     return
 end
 
-function to_matrix(array::DenseAxisArray{T, 1, K}) where {T, K <: NTuple{1, Any}}
+"""
+Convert Vectors, DenseAxisArrays, and SparkAxisArrays to a matrix.
+
+- If the input is a 1d array or DenseAxisArray, the returned matrix will have
```

Comment:
I documented our current logic, but find it confusing. A 5-element vector could be reshaped to a 1x5 or 5x1 matrix. Perhaps we should remove this method and let the caller specify exactly what they want.

Reply:
Defer